app/main.py

```diff
@@ -1,2 +1,41 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.size = 8
+        self.dict_ = [None for i in range(self.size)]
+
+    def resize(self) -> None:
+        temp_ = [el for el in self.dict_ if el is not None]
+        if len(temp_) > (self.size * (2 / 3)):
+            self.size *= 2
+            self.dict_ = [None for _ in range(self.size)]
+            for el in temp_:
+                self.__setitem__(el[0], el[2])
+
+    def __len__(self) -> int:
+        return len([el for el in self.dict_ if el is not None])
```

Comment:
I think it would worth it to create an attribute to store current length so that the dict doesn't have to compute this every time you try to get its length. Especially since that happens every time you add an element...
Oh, wait. You compute the same thing in the `resize()` method even though you have defined the `__len__()`. Well, you shouldn't.